sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/CompressedSource.java

```diff
@@ -319,6 +320,13 @@ public final boolean producesSortedKeys(PipelineOptions options) throws Exceptio
     return sourceDelegate.producesSortedKeys(options);
   }
 
+  @Override
+  public void populateDisplayData(DisplayData.Builder builder) {
+    builder
+        .include(sourceDelegate)
```

Comment:
In `BoundedReadFromUnboundedSource` you included the "source: source.getClass()". Should we do so here as well?

--> We should be consistent as to which one we pick.